config/channels/in-memory-channel/100-config-event-dispatcher.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 The Knative Authors
```

Comment:
the filename should be `config-imc.yaml`. Or `config-in-memory-channel.yaml`.

Reply:
I'd like to keep separate configmap for different components, say we have `config-imc-event-dispatcher` and `config-imc-controller`, this way we can watch only the configmap that the component is interested in.
We can of course have a single `config-imc.yaml` file which potentially holds multiple configmaps. 

What do you think?

Reply:
that makes me think these config parameters are not specific to imc but also to all channels based on the same implementation. So I wonder if instead the configmap should be global and be called `config-delivery.yaml` where all default delivery parameters are set (eg. retries, timeout, etc...)

@vaikas @n3wscott @matzew do you have any opinions on this topic?

Reply:
Is there a use case where different components need different delivery parameters (e.g, timeouts)? Feels like a common use case.

Reply:
If this is not a blocker, perhaps we can defer the decision to wait for more opinions, and meanwhile move forward with this change? I can create an issue to track this.
@lionelvillard What do you think?

Reply:
it's an user-facing choice, so better get this right. The term `delivery` is user-facing, but not `event-dispatcher` which is an internal component. So I suggest to call it `config-delivery.yaml` and to put it under `config`, since it also directly applies to Kafka and potentially to other components.